### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by suhr** [cboxdoerfer/ddb_waveform_seekbar#10]
The last release is more then 1.5 years old.

**Comment by cboxdoerfer** [comment]
Yes, thx for the reminder. Will put together a new release later that day.

**Comment by kpcyrd** [comment]
Last release is from more than 2 years ago :)

**Comment by PF4Public** [comment]
Last release was more than fünfeinhalb years ago
